### PR TITLE
Added feature to display notification reminder

### PR DESCRIPTION
**Pull request by akashdabhi03**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|Yes|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|Yes|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|Yes|
|Is code commenting done properly for the changes made?|Yes|
|Is correct variable naming convention followed?|Yes|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|Yes|
|Are the message strings used in the application taken from strings.xml file?|Yes|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import android.support.v4.app.NotificationCompat;
+
+import com.maq.xprize.onecourse.hindi.R;
+import com.maq.xprize.onecourse.hindi.mainui.MainActivity;
+
+public class NotificationReminderReceiver extends BroadcastReceiver {
+    private static final String CHANNEL_ID = "studyReminderChannel";
+    private static final String LOGTAG = "ReminderNotification";
+    private static final int NOTIFICATION_ID = LOGTAG.hashCode();
+    private static final CharSequence notificationContentText = "It's time to study today's content. Let's start...";
```
Comment (mathewnik90) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java]: Can we get this as well from the strings.xml file?

File: app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            notificationManager.notify(NOTIFICATION_ID, mBuilder.build());
+        } else {
+            mNotificationManager.notify(NOTIFICATION_ID, mBuilder.build());
+        }
+//        }
+    }
+
+    private void createNotificationChannel(Context context) {
+        // Create the NotificationChannel, but only on API 26+ because
```
Comment (mathewnik90) [inline app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java:65]: Similar set of parameters are being set in both mBuilder as well as createNotification. Do you need to do both the steps?

**Review by mathewnik90 — DISMISSED** [review]

File: app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java (hunk 1 above)
Comment (akashdabhi03) [reply app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java]: Yes, we can do that in the next build.

File: app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java (hunk 2 above)
Comment (akashdabhi03) [reply app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java:65]: Both are needed. Notification channel object is used to register a notification channel with system and mBuilder object is used to register a notification with a notification channel.

**Comment by akashdabhi03** [comment]
Added checklist

**Review by mathewnik90 — DISMISSED** [review]

File: app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java (hunk 1 above)
Comment (akashdabhi03) [reply app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java]: Used strings.xml

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**